lib/admin/user.rb

```diff
@@ -20,7 +12,7 @@ def self.authenticate(params = {})
       return nil if username != ENV['USERNAME']
 
       password_hash = Password.new(ENV['PASSWORD_HASH'])
-      User.new(username) if password_hash == params[:password] # The password param gets hashed for us by the == method.
```

Comment:
The comment is almost a duplicate comment of the comment on the bcrypt == method itself.

https://github.com/codahale/bcrypt-ruby/blob/master/lib/bcrypt/password.rb#L64